package.json

```diff
@@ -47,8 +50,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint . --ext .ts,.tsx",
-    "lint:fix": "eslint . --fix --ext .ts,.tsx"
+    "lint": "eslint . --ext .ts,.tsx --max-warnings=0",
```

Comment:
Why `max-warnings=0`? We don't want warnings to trigger non 0 exit code?

Reply:
It's something that I added in the wallet project when we had the issue with the  css variable color animation (if I'm not mistaken). Eventually I'm not sure warnings should make the CI fail, but let's be strict for now. I'll remove it.

Reply:
Done